tern/utils/general.py

```diff
@@ -1,7 +1,7 @@
-'''
-Copyright (c) 2017-2018 VMware, Inc. All Rights Reserved.
-SPDX-License-Identifier: BSD-2-Clause
-'''
+#
+# Copyright (c) 2017-2018 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+#
```

Comment:
Sorry for not noticing this before, but prospector doesn't complain about this string for me. Why change the `''' '''` commit into `#` commits?

Reply:
The `"""` or `'''` leading string is the module document string printed if you were to do a `help(tern.utils.general)` and likely will eventually be pulled into code documentation, e.g., via autodoc.  The copyright and license is not really documentation.  It's a statement about the source file.  This file should have a doc string.  For this cycle of changes, I'm just addressing surface issues.  Makes sense?

Reply:
Yup, makes sense. Didn't realize this file didn't have a doc string. I'll merge it.